### PR TITLE
Increased powershell inline script max length from 5000 to 20000

### PR DESCRIPTION
**Pull request by Chandumca06**
Increased powershell inline script max length from 5000 to 20000

### CODE REVIEW

File: Tasks/PowerShellV2/task.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 2,
-        "Minor": 163,
+        "Major": 3,
```
Comment (jtpetty) [inline Tasks/PowerShellV2/task.json]: Why are you bumping the major version of this task? These seems like a backwards compatible change.

Comment (Chandumca06) [reply Tasks/PowerShellV2/task.json]: I thought initially but now I will change only minor version and update the same. Is that fine?

Comment (Chandumca06) [reply Tasks/PowerShellV2/task.json]: I revert back the major and minor version but I changed only the patch. Let me know is this fine or need to change minor version as well.

Comment (jtpetty) [reply Tasks/PowerShellV2/task.json]: That's fine. Can you rebase your change as well?

Comment (Chandumca06) [reply Tasks/PowerShellV2/task.json]: Done. Please check the recent commit and approve.

**Comment by Chandumca06** [comment]
> LGTM

What to do next? will it merge automatically??

**Comment by jtpetty** [comment]
You have to rebase it, since your change is currently out-of-date with master. Once you do that, I can merge it.

**Comment by Chandumca06** [comment]
> You have to rebase it, since your change is currently out-of-date with master. Once you do that, I can merge it.

Need to create new PR??

**Comment by jtpetty** [comment]
No, not a new PR. Do you see this card on your page?

image

**Comment by Chandumca06** [comment]
> No, not a new PR. Do you see this card on your page?
> 
> image

Yes I clicked on Update Branch...

**Comment by Chandumca06** [comment]
> > No, not a new PR. Do you see this card on your page?
> > image
> 
> Yes I clicked on Update Branch...

No Conflicts so you can merge my changes into master I guess. Please let me know still anything to be done from my end.

**Comment by jtpetty** [comment]
Thanks!

**Comment by Chandumca06** [comment]
> Thanks!

So when can I expect these changes in my azure DevOps pipeline? what to do to reflect these changes in my pipeline?

**Comment by jtpetty** [comment]
@stephenmichaelf - can you comment on this?

**Comment by Chandumca06** [comment]
> @stephenmichaelf - can you comment on this?

Could you please respond asap this change is bit urgent for us.....

**Comment by Chandumca06** [comment]
Hello, when can these changes reflect in production environment? I mean Azure Devops

**Pull request merged**